zap/src/main/java/org/zaproxy/zap/extension/script/ExtensionScript.java

```diff
@@ -1551,6 +1552,29 @@ private void handleScriptException(ScriptWrapper script, Writer writer, Exceptio
         this.setEnabled(script, false);
     }
 
+    /**
+     * Handles errors caused by scripts.
+     *
+     * <p>The given {@code error} will be written to the writer(s) associated with the given
+     * {@code script}, moreover it will be disabled and flagged that has an error.
```

Comment:
`moreover the script will be disabled...`?

Reply:
Tweaked here and in the other method.